lib/install/bin/webpack-dev-server.tt

```diff
@@ -30,6 +39,26 @@ end
 newenv = { "NODE_PATH" => NODE_MODULES_PATH }
 cmdline = [WEBPACK_BIN, "--progress", "--color", "--config", DEV_SERVER_CONFIG] + ARGV
 
+# Trap term and int signals to ensure child processes are removed
+Signal.trap("TERM") do
+  exit 1 if $webpacker_dev_server_pid.nil?
+  # Kill the subprocesses
+  descendant_processes($webpacker_dev_server_pid).each do |d|
+    Process.kill(9, d)
+  end
+end
+
+Signal.trap("INT") do
+  exit 1 if $webpacker_dev_server_pid.nil?
+  # Kill the subprocesses
+  descendant_processes($webpacker_dev_server_pid).each do |d|
+    Process.kill(9, d)
+  end
+end
+
 Dir.chdir(APP_PATH) do
-  exec newenv, *cmdline
```

Comment:
As it's currently using `exec`, the immediate "child" process takes over here. That would seem to make any handling of further processes a matter for that one (the real `webpack-dev-server`). Does that process not manifest the same problem if run directly?